### ISSUE TITLE
about channels and blocks ?

### ISSUE THREAD

**Issue by Jintaek-Rim** [gcp/leela-zero#1869]
Hi.
I want to know about training.
The weight' residual channels are 256 and blocks are 40  now.
(on  September 15, 2018 )
But in src/training/tf/tfprocess.py  channels are 128 and blocks are 6
like below codeline 108, 109.

class TFProcess:
    def __init__(self):
        # Network structure
        self.RESIDUAL_FILTERS = 128
        self.RESIDUAL_BLOCKS = 6

Is it right changing from 128 to 256 and from 6 to 40?

Thanks a lot if you response.

**Comment by Jintaek-Rim** [comment]
It is right.

And in parse.py I should modify like below.

#BATCH_SIZE = 512  
BATCH_SIZE = 64

#RAM_BATCH_SIZE = 128  
RAM_BATCH_SIZE = 16

Because of graphic memory(My graphic card is Nvidia 1050Ti   4G).

But training accuracy is much small.
So I should modify learning rate.
I wonder how much learning rate is proper.
Thanks a lot...

**Comment by alreadydone** [comment]
https://github.com/gcp/leela-zero/issues/1798#issuecomment-418747104
also see bjiyxo's configs below.

**Comment by Jintaek-Rim** [comment]
Thanks a lot to alreadydone....
I follow your advice.
So I adjust like below...

in parse.py
#BATCH_SIZE = 512
BATCH_SIZE = 256

#RAM_BATCH_SIZE = 128
RAM_BATCH_SIZE = 8

in tfprocess.py

        opt = tf.train.MomentumOptimizer(
#            learning_rate=0.05, momentum=0.9, use_nesterov=True)    
            learning_rate=0.0001, momentum=0.9, use_nesterov=True)
